### PR TITLE
implement form component for upload form

### PR DESCRIPTION
**Pull request by lsmith77**
fixes https://github.com/FriendsOfSylius/SyliusImportExportPlugin/issues/24

### CODE REVIEW

**Comment by lsmith77** [comment]
see also https://sylius-devs.slack.com/archives/C3E4BU0QH/p1510409833000020

**Comment by lsmith77** [comment]
@solutionDrive-Alt see the above slack discussion .. what is your point of view here?

**Comment by solutionDrive-Alt** [comment]
What I like about our current solution is the fact, that the form is automatically added to the admin-views, where a Importer is implemented. I can‘t exactly tell, due to my limited experience with sylius/symfony, whether the suggested solutions are better. I am with you, for the fact, that the import-process won‘t be triggered manually over the admin on a daily basis, so the fact that one suggested solution would cause more clicks, wouldn‘t be a problem for me.

**Comment by lsmith77** [comment]
actually I think for now I will just do a hacky solution using a subrequest to render the form. this keeps the PR a bit simpler.

File: src/Form/ImportType.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ImportType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('format', ChoiceType::class, [
+                'choices' => [
+                    'CSV' => 'csv',
```
Comment (lsmith77) [inline src/Form/ImportType.php]: this should eventually be dynamic depending on the resource (ie. country, payment_methods etc).

File: src/Resources/views/Crud/import.html.twig [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{{ render(controller(
```
Comment (lsmith77) [inline src/Resources/views/Crud/import.html.twig]: this is the slightly hacky solution to be able to use the controller to create the form instance

**Comment by lsmith77** [comment]
@solutionDrive-Alt ready for review

File: src/Resources/views/Crud/import.html.twig (hunk 2 above)
Comment (solutionDrive-Alt) [reply src/Resources/views/Crud/import.html.twig]: since I don't know a better solution, it's ok for me...:-)

**Pull request merged**